sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlFlexibleServerPatch.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+{
+    /// <summary> Represents a server to be updated. </summary>
+    public partial class PostgreSqlFlexibleServerPatch
+    {
+        /// <summary> Initializes a new instance of PostgreSqlFlexibleServerPatch. </summary>
+        public PostgreSqlFlexibleServerPatch()
+        {
+            Tags = new ChangeTrackingDictionary<string, string>();
+        }
+
+        /// <summary> The SKU (pricing tier) of the server. </summary>
+        public PostgreSqlFlexibleServerSku Sku { get; set; }
+        /// <summary> Describes the identity of the application. </summary>
+        public PostgreSqlFlexibleServerUserAssignedIdentity Identity { get; set; }
+        /// <summary> Application-specific metadata in the form of key-value pairs. </summary>
+        public IDictionary<string, string> Tags { get; }
+        /// <summary> The location the resource resides in. </summary>
+        public AzureLocation? Location { get; set; }
+        /// <summary> The password of the administrator login. </summary>
+        public string AdministratorLoginPassword { get; set; }
+        /// <summary> PostgreSQL Server version. </summary>
+        public PostgreSqlFlexibleServerVersion? Version { get; set; }
+        /// <summary> Max storage allowed for a server. </summary>
+        public int? StorageSizeInGB
+        {
+            get => Storage is null ? default : Storage.StorageSizeInGB;
+            set
+            {
+                if (Storage is null)
+                    Storage = new PostgreSqlFlexibleServerStorage();
+                Storage.StorageSizeInGB = value;
+            }
+        }
```

Comment:
Same here - others are  not needed in this customized file. Could we remove everything else in this customized file, only keep this property?
BTW please add an attribute on this property: `[EditorBrowsable(EditorBrowsableState.Never)]`
it comes from the namespace `System.ComponentModel`